ezyrb/pod.py

```diff
@@ -28,23 +28,22 @@ class Pod(object):
 		solution file for the computation of the errors. If the solution files
 		does not contain any weight (like volume or area of the cells) the
 		weight is set to 1 for all the cells.
-	:param string namefile_prefix: path and prefix of the solution files.
-		The files are supposed to be named with the same prefix, plus an
+	:param string snapshot_file_regex: path and prefix of the solution files.
+		The files are supposed to be named with	the same prefix, plus an
```

Comment:
I would add here the sentence erased below: For example, in the directory tests/test_datasets/ you can find the files (matlab_00.vtk, matlab_01.vtk, ..., matlab_05.vtk). Or something like this just to be super clear.